OpenSim/Simulation/Model/Model.cpp

```diff
@@ -744,7 +764,7 @@ void Model::extendAddToSystem(SimTK::MultibodySystem& system) const
     Model *mutableThis = const_cast<Model *>(this);
 
     // Ensure Ground is added before all other Components
-    getGround().addToSystem(system);
+    //getGround().addToSystem(system);
```

Comment:
Was this line commented out to allow nested Models (i.e., Ground isn't necessarily at the top of the Component list)? Perhaps delete altogether or add a "TODO" for when nested Models are fully implemented.